sources/packages/backend/apps/db-migrations/src/sql/SFASIndividuals/Add-part-time-msfaa-columns.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE
+    sims.sfas_individuals
+ADD
+    COLUMN part_time_msfaa_number CHAR(10),
+ADD
+    COLUMN part_time_effective_date DATE;
```

Comment:
`part_time_msfaa_effective_date` ? part_time_effective_date seems to be generic

Reply:
It was based on the docs but it makes sense. It is updated now.